zulipterminal/helper.py

```diff
@@ -708,3 +709,36 @@ def get_unused_fence(content: str) -> str:
                                len(max(matches, key=len)) + 1)
 
     return '`' * max_length_fence
+
+
+def get_last_seen_status(timestamp: int) -> str:
+    """
+    Helper to stringify a timestamp value of last seen status.
+
+    * This logic is inspired by `last_seen_status_from_date` in the web app's
+    * `static/js/timerender.js`.
+    """
+
+    minutes = int((datetime.now() - datetime.fromtimestamp(
+                  timestamp)).seconds / 60)
+    if minutes <= 2:
+        return 'Just now'
+    if minutes < 60:
+        return f"{minutes} minutes ago"
+
+    days = (datetime.now() - datetime.fromtimestamp(
+            timestamp)).days
```

Comment:
Re my reply mentioning this code snippet, I think this isn't the right way to get the days. There could be a difference in the `datetime.now()` on L722 and here, and that's simply unreliable. I'll change it to get days by the minutes variable above :+1: